jira-feature-collector/src/main/java/com/capitalone/dashboard/datafactory/jira/JiraDataFactoryImpl.java

```diff
@@ -254,8 +255,20 @@ public String setQuery(String query) {
 				}
 			} catch (Exception e) {
```

Comment:
@amitmawkin , Yes, seriously.  Unfortunately, when claiming the Jira promise in this method, there does not seem to be a way to actually get even the Jira-level exception - please feel free to try if you do not believe me.  The Jira exception propagated here does not seem to be accessible.

You can validate when this exception is thrown by ensuring that a message is never received by the Jira collector from the source system.  However, this is not an artifact of @jimzucker's PR:  Please move this to a new issue in GitHub if you request that it be modified.